README.md

```diff
@@ -15,3 +16,8 @@ the function and run automatically when you commit through a GitHub action.
 ## Runtime Analysis
 
 What is the worst-case big $\Theta$ time complexity of your algorithm?
+
+Worst-case runtime: $\Theta (|E| \cdot |E|!)$ where $E$ is the number of edges in the graph
```

Comment:
As far as I can see you're permuting the nodes, not the edges -- unless I'm missing something?

Reply:
I believe that by permuting the connections for each node it would be changing the edges. For example, in one permutation the first edge could be 0 - 5 and another permutation it could be 0 - 1.

Reply:
But that's what you're doing, isn't it?